x/ccv/provider/keeper/relay.go

```diff
@@ -495,23 +490,17 @@ func (k Keeper) EndBlockCCR(ctx sdk.Context) {
 				)
 				err := k.StopConsumerChain(ctx, channelToChain.ChainId, true)
 				if err != nil {
+					if ccv.ErrConsumerChainNotFound.Is(err) {
+						// consumer chain not found
```

Comment:
It would be great to add here why this can happen

Reply:
see above